CRITICAL - 2021-02-15 13:33:23 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(23): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:34:48 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(23): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:35:40 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(23): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:37:35 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(23): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:46:07 --> syntax error, unexpected 'public' (T_PUBLIC)
#0 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\comple\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:54:05 --> Undefined variable: clave
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(25): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:55:55 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(25): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:57:33 --> El segmento de solicitud de URI estÃ¡ fuera de rango: 3
#0 C:\xampp\htdocs\comple\system\HTTP\URI.php(515): CodeIgniter\HTTP\Exceptions\HTTPException::forURISegmentOutOfRange(3)
#1 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(19): CodeIgniter\HTTP\URI->getSegment(3)
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-15 13:57:48 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(26): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 13:58:19 --> Undefined variable: usuario
#0 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(25): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 25, Array)
#1 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#2 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#3 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-15 13:58:53 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(25): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 14:02:07 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(26): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 14:03:03 --> Trying to get property 'usuario' of non-object
#0 C:\xampp\htdocs\comple\app\Views\Division\ActualizarClave\actualizarClave.php(1): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 1, Array)
#1 C:\xampp\htdocs\comple\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\comple\system\Common.php(1119): CodeIgniter\View\View->render('Division/Actual...', Array, true)
#3 C:\xampp\htdocs\comple\app\Controllers\Division\ActualizarClaveController.php(26): view('Division/Actual...', Array)
#4 C:\xampp\htdocs\comple\system\CodeIgniter.php(918): App\Controllers\Division\ActualizarClaveController->index()
#5 C:\xampp\htdocs\comple\system\CodeIgniter.php(404): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Division\ActualizarClaveController))
#6 C:\xampp\htdocs\comple\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\comple\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-15 20:50:26 --> Maximum execution time of 120 seconds exceeded
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
